packages/foreman/foreman/foreman.spec

```diff
@@ -1169,13 +1170,20 @@ exit 0
 
 %post
 # secret token used for cookie signing etc.
-if [ ! -f %{_datadir}/%{name}/config/initializers/local_secret_token.rb ]; then
+if [ ! -e %{_datadir}/%{name}/config/initializers/local_secret_token.rb ] ; then
   touch %{_datadir}/%{name}/config/initializers/local_secret_token.rb
   chmod 0660 %{_datadir}/%{name}/config/initializers/local_secret_token.rb
   chgrp foreman %{_datadir}/%{name}/config/initializers/local_secret_token.rb
   %{foreman_rake} security:generate_token >/dev/null 2>&1 || :
   chmod 0640 %{_datadir}/%{name}/config/initializers/local_secret_token.rb
 fi
+if [ -f %{_datadir}/%{name}/config/initializers/local_secret_token.rb ] ; then
```

Comment:
In 796febe4203eb87f03f24a4d6a223093a1f2f786 I fixed an oversight I had: `-f` dereferences symlinks.